### ISSUE TITLE
TypeLoadException after updating to Xamarin.Forms.3.3.0

### ISSUE THREAD

**Issue by felipecds** [xamarin/Xamarin.Forms#4197]
### Description
Unhandled Exception:

System.TypeLoadException: Could not resolve type with token 010000cd (from typeref, class/assembly Xamarin.Forms.TextDecorations, Xamarin.Forms.Core, Version=2.0.0.0, Culture=neutral, PublicKeyToken=null) occurred

when I set TextDecorations on a Span via Custom Converter. Cleaned all folders, rebuild, build, close IDE, reopened

- Version with issue: 3.3.0.912540
- IDE: Visual Studio 2017
- Platform Target Frameworks: 
  - iOS:  11.4
  - Android: API 27

**Comment by samhouts** [comment]
@felipecds Can you please attach a small reproduction project that exhibits the issue? Thanks!

**Comment by felipecds** [comment]
@samhouts Of course, just need to be home later today. I'll reply this when I post the sample.

**Comment by nwikechisom** [comment]
Hello @PureWeen, I am experiencing the same issue on this app [here](https://dev.azure.com/iamoge/JED)
<img width="1680" alt="typeloadexception" src="https://user-images.githubusercontent.com/26902155/72883764-2ef28d00-3d05-11ea-9d12-b736e303a39d.png">
